Rocket.Chat/External/RCEmojiKit/Views/EmojiPicker/EmojiPicker.swift

```diff
@@ -284,17 +284,19 @@ extension EmojiPicker: UICollectionViewDelegateFlowLayout {
 
 extension EmojiPicker: UITabBarDelegate {
     func tabBar(_ tabBar: UITabBar, didSelect item: UITabBarItem) {
-        guard let index = tabBar.items?.index(of: item) else { return }
+        if !self.emojisCollectionView.isDragging {
```

Comment:
Could you make this check in a guard also?

Reply:
Of course. I will change it

Reply:
I've changed it